### ISSUE TITLE
lxd: called with unexpected argument 'nftablesSupport'

### ISSUE THREAD

**Issue by raboof** [NixOS/nixpkgs#144420]
### Describe the bug

Rebuilding my system with a more recent nixos-unstable, I get:

```
error: anonymous function at /home/aengelen/nixpkgs/pkgs/tools/admin/lxd/default.nix:1:1 called with unexpected argument 'nftablesSupport'
```

### Steps To Reproduce
1. enable lxc and lxd in /etc/nixos/configuration.nix
2. set nixpkgs to nixos-unstable (as of b67e752c29f18a0ca5534a07661366d6a2c2e649)
3. `nixos-rebuild switch`

**Comment by cid-chan** [comment]
Introduced by this commit:
https://github.com/NixOS/nixpkgs/commit/b112f366fb6853ca8582cdef2def20acc4717af5

Apparently, nftables rae the default now. You should be able to safely remove the `nftablesSupport = true` override.

**Comment by raboof** [comment]
Thanks, indeed - this was already fixed on master with #144362
